playbooks/bootstrap.yml

```diff
@@ -59,3 +59,9 @@
         tasks_from: ksm.yml
       tags:
         - edpm_kernel
+    - name: Configure hugepages
+      ansible.builtin.import_role:
+        name: osp.edpm.edpm_extra_mounts_default
+        tasks_from: edpm_extra_mounts_default.yml
+      tags:
+        - edpm_extra_mounts_default
```

Comment:
Hm, it feels strange that the task is specific about huge pages while the role name is generic and talking about any extra mounts, but then also just mounts the huge pages.